app/src/main/java/org/mozilla/reference/browser/ext/TestConfig.kt

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.reference.browser.ext
+
+import android.os.Bundle
+
+var testConfig: Bundle? = null
```

Comment:
Can we attach this to something (maybe to the `EngineProvider`)? It's a bit weird to have this global `var` :)

Reply:
Yeah, kept this separate, so we can extends for `WebView` testing where having it accessible in the `EngineProvider` isn't enough. However, since we haven't decided how to best handle `WebView` testing, I will move this over and we can decide later.